gstools/field/generator.py

```diff
@@ -10,16 +10,23 @@
    RandMeth
    IncomprRandMeth
 """
-# pylint: disable=C0103, W0222
+# pylint: disable=C0103, W0222, C0412
 import warnings
 from copy import deepcopy as dcp
 
 import numpy as np
 
+from gstools import config
 from gstools.covmodel.base import CovModel
-from gstools.field.summator import summate, summate_incompr
 from gstools.random.rng import RNG
 
+if config.USE_RUST:
+    # pylint: disable=E0401
+    from gstools_core import summate, summate_incompr  # pragma: no cover
+else:
+    # pylint: disable=C0412
+    from gstools.field.summator import summate, summate_incompr
```

Comment:
What about setting `# pragma: no cover` behind `if config.USE_RUST:`?
Also the `# pylint: disable=C0412` is redundant (see line 13) and we could also just add `E0401` in line 13.